homeassistant/components/netgear/__init__.py

```diff
@@ -18,6 +22,21 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except CannotLoginException as ex:
         raise ConfigEntryNotReady from ex
 
+    port = entry.data.get(CONF_PORT)
+    ssl = entry.data.get(CONF_SSL)
+    if port != router.port or ssl != router.ssl:
+        data = entry.data.copy()
+        data[CONF_PORT] = router.port
+        data[CONF_SSL] = router.ssl
+        hass.config_entries.async_update_entry(entry, data=data)
+        _LOGGER.warning(
```

Comment:
Shouldn't be a warning since it will continue to work
```suggestion
        _LOGGER.info(
```